app/models/feedback_message.rb

```diff
@@ -4,6 +4,8 @@ class FeedbackMessage < ApplicationRecord
   belongs_to :offender, class_name: "User", optional: true, inverse_of: :offender_feedback_messages
   belongs_to :reporter, class_name: "User", optional: true, inverse_of: :reporter_feedback_messages
   belongs_to :affected, class_name: "User", optional: true, inverse_of: :affected_feedback_messages
+
+  has_one :email_message, dependent: :nullify
```

Comment:
If i understood the code correctly a feedback message can have only one optional associated email message

Reply:
What do we use these EmailMessages for? Is there a reason to keep them around even if the feedback message that created it is destroyed?

Reply:
AFAIK this is used for conflict resolution purposes. In case a violation occurs we send an email. I think for audit purposes we should keep a copy of such email